tests/MoBi.Tests/OSPSuiteCore.Tests/CoreExpressionProfileSpecs.cs

```diff
@@ -0,0 +1,44 @@
+﻿using OSPSuite.BDDHelper;
+using OSPSuite.BDDHelper.Extensions;
+using OSPSuite.Core.Domain.Builder;
+
+namespace MoBi.OSPSuiteCore.Tests
+{
+   public class concern_for_CoreExpressionProfile : ContextSpecification<CoreExpressionProfile>
```

Comment:
Most of the stuff in these objects are just setting properties. So I'm not testing it.